pkg/athena/materializer.go

```diff
@@ -0,0 +1,65 @@
+package athena
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bruin-data/bruin/pkg/helpers"
+	"github.com/bruin-data/bruin/pkg/pipeline"
+)
+
+// The other packages all use a materializer that renders the query to a single string. Due to the quirks of athena
+// we need to create a different materializer that returns a slice of strings, since databricks server requires us to send separate batches
```

Comment:
```suggestion
// we need to create a different materializer that returns a slice of strings, since athena server requires us to send separate batches
```